CHANGELOG.md

```diff
@@ -105,6 +105,11 @@ For older changes see the [archived Singularity change log](https://github.com/a
   configuration file being written.
 - `--no-https` now applies to connections made to library services specified
   in `library://<hostname>/...` URIs.
+- Correct documentation for sign command r.e. source of key index.
```

Comment:
Please delete this line, it is not in original PR

Reply:
This line from sylabs/singularity#469. I included it to description.